backend/app/api/api_v1/schemas/document.py

```diff
@@ -106,10 +106,44 @@ class DocumentAssociationCreateRequest(BaseModel):
     type: str
 
 
+# TODO: BAK-1137 Remove
 class DocumentAssociationCreateResponse(DocumentAssociationCreateRequest):
     """Schema for associations create response."""
 
     id: int
 
     class Config:  # noqa: D106
         orm_mode = True
+
+
+class RelationshipCreateRequest(BaseModel):
+    """Schema for Relationship create request."""
+
+    name: str
+    type: str
+    description: str
+
+
+class RelationshipEntityResponse(RelationshipCreateRequest):
+    """Response for Relationship create request."""
+
+    id: int
+
+    class Config:  # noqa: D106
+        orm_mode = True
+
+
+class RelationshipGetResponse(BaseModel):
+    """Response for Relationship get request."""
+
+    relationships: List[RelationshipEntityResponse]
```

Comment:
This doesn't look right to me... if we're getting a list of relationships can we not just return an array at the top level? I think this is what we've done elsewhere. If that's not "correct" and we want all responses to be a JSON object at the top level we should raise a ticket to fix the other response types...

Reply:
We probably need to chat about this - the main reason this exists is to pull the object from the orm ... Meanwhile, I will follow the pattern for https://github.com/climatepolicyradar/navigator/blob/0f1c2a4f77963bd2eb4081d4d54e73de9906be98/backend/app/api/api_v1/routers/admin.py#L32-L48